pkg/lib/saml/samlprotocol/update_helper.go

```diff
@@ -0,0 +1,15 @@
+//go:build updatecrewjamsaml
+
+package samlprotocol
+
+// The intended usage of this file is to replace the file in the repository with the original file.
+// So that you can use `git diff` to view the changes.
+// This file is guarded by a build tag so it does not run normally.
+//
+// It is intended you run it with
+//  go generate -tags updatecrewjamsaml ./pkg/lib/saml/samlprotocol
+
+//go:generate curl -sSL https://raw.githubusercontent.com/crewjam/saml/refs/tags/v0.5.0/schema.go -o schema.go
+//go:generate curl -sSL https://raw.githubusercontent.com/crewjam/saml/refs/tags/v0.5.0/duration.go -o duration.go
+//go:generate curl -sSL https://raw.githubusercontent.com/crewjam/saml/refs/tags/v0.5.0/time.go -o time.go
+//go:generate curl -sSL https://raw.githubusercontent.com/crewjam/saml/refs/tags/v0.5.0/metadata.go -o metadata.go
```

Comment:
I think it is not intended to sync the files.
The reason we version the files in our repo is we will change them, so it might not be same as the original file.

Reply:
The intention is not to sync them, but to review the changes. So this file does not run by default.